101-vm-ubuntu-DSVM-GPU-or-CPU/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "Deploy a Ubuntu Linux DataScience VM 16.04-LTS.",
+    "description": "This template deploy a Ubuntu Server with some tools for Data Science. You can provide the username, password, virtual machine name and select between CPU or GPU computing.",
+    "summary": "This template takes a minimum amount of parameters and deploys a DataScience Linux VM.",
+    "githubUsername": "leestott",
+    "dateUpdated": "2019-11-11",
+    "type": "QuickStart",
+    "tags": {
+        "subscriptiontype": "Azure4Student",
+        "resource": "vm, datascience, DSVM",
+        "os": "Linux, Ubuntu",
+        "level": "beginner",
+        "cost": "$$"
```

Comment:
GPU is going to be a higher cost?

Reply:
Yes ... It will cost almost double. Should I put it as $$$?

Reply:
so the cost of this could be $$ or $$$$?  Do we handle a range (e.g. 

"cost": "$$-$$$$"

?